assessment/report_generator.go

```diff
@@ -66,6 +67,100 @@ func GenerateReport(dbName string, assessmentOutput utils.AssessmentOutput) {
 	logger.Log.Info("completed publishing sample report")
 }
 
+func writeNonSchemaChanges(dbName string, nonSchemaChanges [][]string) {
+	f, err := os.Create(dbName + "_non_schema_changes.txt")
+	if err != nil {
+		logger.Log.Error(fmt.Sprintf("Can't create schema file %s: %v", dbName, err))
+		return
+	}
+	defer f.Close()
+
+	w := csv.NewWriter(f)
+	w.Comma = '|'
+	w.UseCRLF = true
+
+	w.WriteAll(nonSchemaChanges)
+	logger.Log.Info("completed publishing non schema changes report")
+}
+
+func writeRawChanges(dbName string, snippets []utils.Snippet) {
```

Comment:
it would help for this method to have a clearer name - in this case i think it is raw code comparison